### ISSUE TITLE
Application fails when using slashes in its name

### ISSUE THREAD

**Issue by kewinrausch** [IRATI/stack#368]
<a href="https://github.com/sandervrijders"><img src="https://avatars.githubusercontent.com/u/1212456?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [sandervrijders](https://github.com/sandervrijders)**
_Monday Sep 01, 2014 at 14:25 GMT_
_Originally opened as https://github.com/dana-i2cat/irati/issues/365_

----

When an application uses slashes in its ApName (or any other name), and it registers, it is not disseminated in the DIF since the RIB uses slashes internally as a separator. The dissemination fails because of this. 

Temporary workaround: Do not uses slashes in your application names.

**Comment by sandervrijders** [comment]
@msune Could this problem be solved with the new RIB?

**Comment by msune** [comment]
No... this is librina. Please @sandervrijders stop assigning people randomly and milestones; milestone 1.4.0 is **not** unscheduled, it means strictly it will be implemented in the next release, which has to be discussed first

**Comment by sandervrijders** [comment]
@msune Ok wrt to MS. I am assigning the maintainers so they can keep track of the issue. Or in this case, the bug existed because of the RIB.
